style.css

```diff
@@ -613,7 +613,7 @@ section .location {
     margin-left: 1.5em;
   }
   #resume {
-    width: 820px;
+    width: 900px;
```

Comment:
how it that related with supporting highlights in Education?

Reply:
Ignore this change, it was a mistake.